util/util.go

```diff
@@ -20,7 +21,10 @@ import (
 // the current OS/ARCH instead of all possible permutations.
 // This is not concurrently safe which is ok for the current application. If
 // other needs arise it may need to be re-written.
+const DefaultGoGetTimeout time.Duration = time.Second * 5
+
 var ResolveCurrent = false
+var GoGetTimeout = DefaultGoGetTimeout
```

Comment:
Please name the variable something related to its purpose. The provenance of this timeout is not relevant when it comes to variable naming.